zerver/lib/test_classes.py

```diff
@@ -2296,6 +2296,7 @@ class WebhookTestCase(ZulipTestCase):
     CHANNEL_NAME: str | None = None
     TEST_USER_EMAIL = "webhook-bot@zulip.com"
     URL_TEMPLATE: str
+    LEGACY_URL_TEMPLATES: list[str] | None = None
```

Comment:
Many webhooks use URL_TEMPLATE, so created a separate variable for legacy templates.